llvm/lib/Analysis/IR2Vec.cpp

```diff
@@ -518,7 +518,8 @@ IR2VecVocabAnalysis::run(Module &M, ModuleAnalysisManager &AM) {
   // Otherwise, try to read from the vocabulary file.
   if (VocabFile.empty()) {
     // FIXME: Use default vocabulary
-    Ctx->emitError("IR2Vec vocabulary file path not specified");
+    Ctx->emitError("IR2Vec vocabulary file path not specified; You may need to "
```

Comment:
This doesn't really have anything to do with the patch (based on the description) and shouldn't really be here, but separating it out is probably too much of a hassle for what it is. Leaving it should be fine.